### PR TITLE
Sort out svg shenanigans

### PR DESCRIPTION
**Pull request by louis-bompart**
Simple PR, just noticed that the icon import was flagged in AttachedResult.ts
I chose to use the utils instead of including the ref typings for the svg module.

### CODE REVIEW

File: config/tsconfig.json [hunk 1]
```diff
@@ -16,6 +16,6 @@
         "typeRoots": ["../src/typings"],
         "types": ["jasmine", "coveo-search-ui", "coveo-search-ui-tests"]
     },
-    "include": ["../src/**/*.ts", "../src/**/*.svg", "../src/**/*.json"],
+    "include": ["../src/**/*.ts", "../img/**/*.svg", "../src/**/*.json"],
```
Comment (jeremierobert-coveo) [inline config/tsconfig.json]: Je suis currieux de vois le code ressemble a quoi une fois transpiller ?

File: src/utils/icons.ts [hunk 2]
```diff
@@ -5,8 +5,11 @@ import * as DUPLICATE from '../../img/duplicate.svg';
 import * as SEARCH from '../../img/search.svg';
 import * as VIEW from '../../img/view.svg';
 import * as DOT from '../../img/dot.svg';
+import * as PAPER_CLIP from '../../../img/paperclip.svg';
```
Comment (jeremierobert-coveo) [inline src/utils/icons.ts]: ../.. et non ../../..

File: config/tsconfig.json (hunk 1 above)
Comment (louis-bompart) [reply config/tsconfig.json]: `this.buttonElement = Object(coveo_search_ui__WEBPACK_IMPORTED_MODULE_0__["$$"])('div', {}, _utils_icons__WEBPACK_IMPORTED_MODULE_1__["paperclipIcon"]).el;`
`this.buttonElement = Object(coveo_search_ui__WEBPACK_IMPORTED_MODULE_0__["$$"])('div', {}, _img_paperclip_svg__WEBPACK_IMPORTED_MODULE_1__).el;`

The webpack shenanigans behind are pretty similar, just the hoop through utils before the svg markup as a string.

Comment (jeremierobert-coveo) [reply config/tsconfig.json]: Good 👍

**Pull request merged**